### PR TITLE
Add mixpanel to emulator

### PR DESCRIPTION
**Pull request by sukantoraymond**
Add user tracking to emulator

______

For contributor use:

- [x] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: internal/emulator/start.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ func ConfiguredServiceKey(
 	var state *flowkit.State
 	var err error
 	loader := &afero.Afero{Fs: afero.NewOsFs()}
+	command.HandleUserTracking(Cmd, loader)
```
Comment (sukantoraymond) [inline internal/emulator/start.go:56]: need to add this since emulator is not part of addtoparent

File: internal/command/command.go [hunk 2]
```diff
@@ -78,7 +78,7 @@ const (
 	logLevelNone  = "none"
 )
 
-func (c Command) handleUserTracking(loader flowkit.ReaderWriter) {
+func HandleUserTracking(command *cobra.Command, loader flowkit.ReaderWriter) {
```
Comment (sideninja) [inline internal/command/command.go:81]: please move this to the bottom where helpers are. I would also rename it to: `RecordCommandUsage`

Comment (sukantoraymond) [reply internal/command/command.go]: addressed

**Comment by sideninja** [comment]
Have you manually tested this usage of emulator and other commands?

**Comment by sukantoraymond** [comment]
Already tested, they are being tracked in mixpanel

**Review by sideninja — APPROVED** [review]

**Pull request merged**